submissions/DeamonFire/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="css/style.css">
+        <title>Hooli-style Popup</title>
+    </head>
+    <body>
+        <header class="header">
+            <h1 class="header__h1">Hooli-style Popup</h1>
+            <nav class="header__nav">
+                <ul class="header__menu">
+
+                    <li class="header__menu_block">
+                        <a href="" class="header__menu_block_link">Main</a>
+                    </li>
+
+                    <li class="header__menu_block">
+                        <a href="" class="header__menu_block_link">Images</a>
+                    </li>
+
+                    <li class="header__menu_block">
+                        <div class="header__menu_block_hidden popup-menu">
+                            <input type="checkbox" id="menu_1" class="popup-menu__input">
+                            <label for="menu_1" class="popup-menu__label" aria-label="Advanced menu"></label>
+                            <div class="popup">
+
+                                    <ul class="popup_hidden wrapper">
+                                        <li class="wrapper__block">
+                                            <a href="" class="wrapper__block_link">
+                                                <div class="wrapper__block_link_item">
+                                                    <img src="popup-icons/calendar.png" class="wrapper__block_img">
+                                                    <div class="wrapper__block_title">Calendar</div>
+                                                </div>
+                                            </a>
+                                        </li>
+                            
+                                        <li class="wrapper__block">
+                                            <a href="" class="wrapper__block_link">
+                                                <div class="wrapper__block_link_item">
+                                                    <img src="popup-icons/mail.png" class="wrapper__block_img">
```

Comment:
The `alt` is the required attribute for `img` element, if it's empty just add it with empty value
```HTML
<img alt="" >
```

Reply:
Fixed that as well